C_CPP/Data Structures/Queue/Queue Implementation/readme.md

```diff
@@ -1 +1,38 @@
+# QUEUE
+
+Queue is a linear data structure which follows a particular order in which operations are performed i.e. **First in First Out** also known as **FIFO**.
+FIFO principle implies that the first element inserted will be the first one to be removed.<br>
+Real life example of queue can be,line of vehicles on road during heavy traffic.Here the vehicle which came first will be the one to pass the traffic.<br><br>
+<img src="https://github.com/ishrutik/winter-of-contributing/blob/C_CPP/C_CPP/Data%20Structures/Queue/Queue%20Implementation/Queue_Real_Life_Example.jpg"/>
+<br><br><br>
+Queue can be implemented using both linked list and arrays.<br><br>
+Implementation using Array<br>
+[Queue Implementation using Array](https://github.com/ishrutik/winter-of-contributing/blob/C_CPP/C_CPP/Data%20Structures/Standard%20Template%20Library%20(Vectors%2C%20Maps%20etc)/Queue%20Implementation/Queue_Using_Array.cpp)
+<br><br>
+Implementation using Linked List<br>
+[Queue Implementation using Linked List](https://github.com/ishrutik/winter-of-contributing/blob/C_CPP/C_CPP/Data%20Structures/Standard%20Template%20Library%20(Vectors%2C%20Maps%20etc)/Queue%20Implementation/Queue_Using_LL.cpp)

```

Comment:
It says page not found 
![2021-10-05 (3)](https://user-images.githubusercontent.com/75165587/136027334-c81807f0-e408-460a-9595-6a8116330e39.png)